app/assets/javascripts/listings/components/eligibilitySection.js.coffee

```diff
@@ -12,8 +12,10 @@ angular.module('dahlia.components')
       @error = ListingPreferenceService.error
 
       @occupancy = (unitSummary) ->
-        return '1' if unitSummary.maxOccupancy == 1
-        unitSummary.minOccupancy + '-' + unitSummary.maxOccupancy
+        if unitSummary.minOccupancy == 1 || unitSummary.maxOccupancy == null
+          "at least #{unitSummary.minOccupancy}"
+        else
+          "#{unitSummary.minOccupancy}-#{unitSummary.maxOccupancy}"
 
       @occupancyLabel = (maxOccupancy) ->
         return $translate.instant('LISTINGS.PERSON') if maxOccupancy == 1
```

Comment:
looks like this is what you can update to have it say person instead of people

Reply:
@akegan @occupancyLabel is being used in multiple places and used in other ways which needs this to stay the way it is, and so I just updated @occupancy for this use case.

Reply:
oh I see. I think that we should update the function to make that clearer since it's not always maxOccupancy that's going in there.